### DISCUSSION TITLE
M2O Viewable but not Editable?

### DISCUSSION THREAD

**Discussion by ldrach** [directus/directus#13016] [Archive]
Hello,

I'm pretty new to Directus and I've been looking through the discussion to see if anyone was having a similar issue to me but did not have any luck.

For context, I'm working on a setup for registration forms. There will be an Applicant role that will fill out the form and then a Clerical role will review the form and approve or reject the application. It's a fairly large form so I made multiple collections for the different sections (Address, Student, etc) and then made M2O relationships to each of the different collections needed for the form from a parent "Registration Form" collection.

The goal is to set up the permissions so that when the Clerical role looks at the Registration Form collection, they can view all of the M2O relationships as well as click into the field to view all of the child fields but cannot edit them. They will only have access to one drop down field in the Registration Form collection used for approving or rejecting the application.

The problem I am having is that if I only allow the Clerical role to view the fields, they cannot enter the M2O relationship to view the data being entered, they can only see the display template for that field.

![image](https://user-images.githubusercontent.com/40499729/165340651-6a55d885-a28b-4d9e-97a7-2224ea5a697a.png)

I need the Clerical role to be able click into something like the Address collection and verify the data without allowing them to edit the item or any of the fields in it like below.

![image](https://user-images.githubusercontent.com/40499729/165341989-e410b054-f55e-4a80-8651-693e656eaf51.png)

I was hoping to set the permissions for Registration Form so that it was only viewable with the drop down for approval being updatable; however, this would make the items unclickable like the first picture. Giving permission to edit the Address or Student item lets the role click into the relationship to view the data in the collection but still allows the role to select which row in the collection to associate with the parent Registration Form.

![image](https://user-images.githubusercontent.com/40499729/165341756-dd8d8e47-a690-4138-b097-90ef68ee72c4.png)

From the testing I have done, it looks like the exact permission setup I'm looking for may not be possible but I thought it was worth making a post about it just in case. 

Thanks!
Lyndon

**Comment by rijkvanzanten** [comment]
This was just added as part of https://github.com/directus/directus/pull/13207 🙂
